### ISSUE TITLE
Model Finetuning D4 -> PDBBind

### ISSUE THREAD

**Issue by zgaleday** [keiserlab/torch_pgn#14]
As discussed in the meeting I applied the D4 pretrained model to pdbbind. To set a baseline with the setup for reference, training the pdbbind model from scratch yields the following results on the validation set:
![image](https://user-images.githubusercontent.com/8671930/148154405-1472757b-20ad-4719-b598-913c365f78e0.png)

As a first pass I just used the D4 pretrained model as a starting point for regular training. This resulted in slightly decrease performance.

![image](https://user-images.githubusercontent.com/8671930/148157045-3d46dab9-335d-4fd8-9818-5d2941ae9a5b.png)

After this, I took the pretrained D4 model and froze the encoder weights, while allowing the FF networks to train. This resulted in plateaued training (no overfitting) with worse performance. 

![image](https://user-images.githubusercontent.com/8671930/148154526-e040ef87-fabb-4853-8617-0da0a4faa7f2.png)

Since the model appear to be stable I wondered if unfreezing the encoder weights once the FF network was trained would improve preformance. There was minimal effect with overfitting occuring soon after the encoder was allowed to train (see below).

![image](https://user-images.githubusercontent.com/8671930/148154714-d4db81b4-b307-44cd-9e8b-68dba18d7158.png)
![image](https://user-images.githubusercontent.com/8671930/148154698-48b49a6b-cd2a-4cbb-9848-d00f0653ee56.png)

Seems like starting from D4 parameter decreases model preformance
